manager/apis/app/v1alpha1/m4dmodule_types.go

```diff
@@ -74,6 +74,25 @@ type SupportedAction struct {
 	Level connectors.EnforcementAction_EnforcementActionLevel `json:"level,omitempty"`
 }
 
+// The spec of the endpoint exposed by the module
+type EndpointSpec struct {
+	// +optional
+	Hostname string `json:"hostname"`
```

Comment:
I intended to use it, you and Florian thought it's redundant.. Anyhow this field is used currently by the status of the m4d-application. I'll add comment.

Reply:
Then I guess I was wrong or misunderstood, happens. Anyway, we don't dictate that the service would be named just as the release name. It has 10 more letters to use so it can append for example "-service" to the release name in the Helm template. But I'm okay with forcing it to be just the release name until it will hit us again, you should add it to the module docs though.